client/db.go

```diff
@@ -252,78 +250,68 @@ func (db *DB) GetProto(key interface{}, msg gogoproto.Message) error {
 	if err != nil {
 		return err
 	}
-	return r.Rows[0].ValueProto(msg)
+	return r.ValueProto(msg)
 }
 
-// Put sets the value for a key.
-//
-// The returned Result will contain a single row and Result.Err will indicate
-// success or failure.
+// Put sets the value for a key, returning the set key/value or an error.
 //
 // key can be either a byte slice, a string, a fmt.Stringer or an
 // encoding.BinaryMarshaler. value can be any key type or a proto.Message.
-func (db *DB) Put(key, value interface{}) (Result, error) {
-	return runOne(db, db.B.Put(key, value))
+func (db *DB) Put(key, value interface{}) (KeyValue, error) {
```

Comment:
Yeah, good point. We never actually used the return value of `Put/CPut/Del`. Removed.